avocado/utils/network/common.py

```diff
@@ -0,0 +1,12 @@
+from .. import process
+
+
+# Probably this will be replaced by aexpect
+def run_command(command, host, sudo=False):
+    # This is used to avoid circular imports
+    if host.__class__.__name__ == 'LocalHost':
```

Comment:
I'd avoid calling instance dunders directly... you can import `LocalHost` and use `isinstance()` or `type()` to compare `host` against.

Reply:
If I use this here, I will have errors because of circular imports.

Reply:
True. It looks like `run_command` should be a method implemented by each `Host`'s children. Anyway, no worries, just leave it.